app/src/main/res/values/strings.xml

```diff
@@ -436,15 +438,15 @@
     <string name="new_version_date_colon">"New Version date: "</string>
     <string name="old_version_date_colon">"Old Version date: "</string>
     <string name="speak_your_note_text">Speak your note text</string>
-    <string name="alarm_if_above_high_value">Alarm if above high value</string>
+    <string name="alarm_if_above_high_value">Notify if above threshold and not falling for long</string>
```

Comment:
I don't think this english is clear what this means

Reply:
How about this instead?  

*Notify if above threshold for longer than the specified time below*

Reply:
I have changed the string to something that I hope is more clear.

Thanks for the review